domains/certificates/Query.API/API.AppTests/SwaggerTests.cs

```diff
@@ -9,11 +10,18 @@
 namespace API.AppTests;
 
 [UsesVerify]
-public class SwaggerTests : IClassFixture<QueryApiWebApplicationFactory>
+public class SwaggerTests : IClassFixture<QueryApiWebApplicationFactory>, IClassFixture<RabbitMqContainer>
```

Comment:
Is the RabbitMqContainer needed in order for the swagger tests to work? If yes, is that due to the WaitUntilStarted trick we did in QueryApiWebApplicationFactory ?

Reply:
Yes else it will keep waiting